### PR TITLE
testing/lightdm-gtk-greeter: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://launchpad.net/lightdm-gtk-greeter
Gtk based greeter for lightdm.

### CODE REVIEW

File: testing/lightdm-gtk-greeter/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=lightdm-gtk-greeter
+pkgver=2.0.5
+pkgrel=0
+pkgdesc="Gtk based greeter for lightdm."
+url="https://launchpad.net/lightdm-gtk-greeter"
+arch="all"
+license="GPL-3.0-or-later"
```
Comment (TBK) [inline testing/lightdm-gtk-greeter/APKBUILD]: If there is no test suite then please add `options="!check" # No test suite`

**Comment by ncopa** [comment]
can you please merge the 2 commits and rebase?

You can run `git rebase -i`, write `f` for "fixup" on the second commit, and it will merge into the first.

**Comment by tanertas** [comment]
Done.

File: testing/lightdm-gtk-greeter/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+depends="lightdm"
+makedepends="gtk+3.0-dev lightdm-dev exo-dev"
+subpackages="$pkgname-lang $pkgname-doc"
+source="https://launchpad.net/${pkgname}/${pkgver%.?}/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz"
+options="!check"
+
+build() {
+	cd "$builddir"
+	  ./configure \
```
Comment (ncopa) [inline testing/lightdm-gtk-greeter/APKBUILD]: whitespace damage here. there is a couple of spaces before `./configure`

Comment (ncopa) [reply testing/lightdm-gtk-greeter/APKBUILD]: actually, I'll just fix it for you.

Comment (tanertas) [reply testing/lightdm-gtk-greeter/APKBUILD]: Thanks. Your eyes sharper than mine.

**Pull request closed without merging**